qa/test_images/image_list.txt

```diff
@@ -0,0 +1,13 @@
+410.jpg
```

Comment:
Since this is now enumerated in a separate file, would it be worth having https://github.com/NVIDIA/DALI/blob/master/dali/test/dali_test_single_op.h#L38 import this list instead of hard-coding the same list?

Reply:
Done